infrastructure/variables.tf

```diff
@@ -37,13 +31,20 @@ variable "subnetwork_name" {
 }
 
 variable "subnetwork_cidr" {
-  type    = string
-  default = "10.128.0.0/20"
+  type        = string
+  default     = "10.128.0.0/16"
```

Comment:
is this just random?

Reply:
This is based on the default block for auto subnet. Added a comment.